test/functional/signrawtransactions.py

```diff
@@ -34,6 +34,16 @@ def successful_signing_test(self):
         outputs = {'mpLQjfK79b7CCV4VMJWEWAj5Mpx8Up5zxB': 0.1}
 
         rawTx = self.nodes[0].createrawtransaction(inputs, outputs)
+        rawTxSigned = self.nodes[0].signrawtransactionwithkey(rawTx, privKeys, inputs)
+
+        # 1) The transaction has a complete set of signatures
+        assert 'complete' in rawTxSigned
```

Comment:
Remove key asserts since below you assert the value, which fail in case the key is not defined? cc @jnewbery.

Reply:
Right. The line below should raise KeyError in that case.